teku/src/main/java/tech/pegasys/teku/AbstractNode.java

```diff
@@ -124,13 +135,29 @@ public void start() {
     metricsEndpoint.start().join();
     metricsPublisher.start().join();
     getServiceController().start().join();
+    counterMaintainer =
+        Optional.of(
+            serviceConfig
+                .createAsyncRunner("RejectedExecutionCounter", 1)
+                .runWithFixedDelay(
+                    this::pollRejectedExecutions,
+                    Duration.ofSeconds(5),
+                    (err) -> LOG.debug("rejected execution poll failed", err)));
+  }
```

Comment:
not sure if there's a prettier way of doing this at this point, didn't seem to be anything obvious...